### PR TITLE
Backport 1211 to 7.x.x

### PR DESCRIPTION
**Pull request by fredvd**
Make masking specific validation errors configurable in DX DeserializeFromJson.

### CODE REVIEW

**Comment by fredvd** [comment]
There’s only 2 minor test problems, caused by a recent black update and the bleach package no longer being pinned on python2.7 in some combinations.  Only 5.1 still failes.

File: versions.cfg [hunk 1]
```diff
@@ -2,6 +2,15 @@
 # Buildout
 setuptools =
 zc.buildout =
+plone.restapi =
```
Comment (mauritsvanrees) [inline versions.cfg:5]: The empty `plone.restapi` pin should not be needed: this is done in `base.cfg` already.

**Review by mauritsvanrees — APPROVED** [review]
One minor comment, but LGTM. Thanks!

**Comment by mauritsvanrees** [comment]
@plone/restapi-team A release of 7.x.x would be great for Plone 5.2.8, which is [pending](https://community.plone.org/t/plone-5-2-8-soft-released/15230).

**Comment by mauritsvanrees** [comment]
No reaction, so I have released 7.8.0 myself.

**Pull request merged**